pkg/sdk/symbols/initialize/initialize.go

```diff
@@ -101,14 +101,19 @@ func plugin_init(config *C.char, rc *int32) C.uintptr_t {
 		*rc = sdk.SSPluginSuccess
 	}
 
-	return (C.uintptr_t)(cgo.NewHandle(state))
+	handle := cgo.NewHandle(state)
+	if *rc == sdk.SSPluginSuccess {
+		hooks.OnAfterInit()(handle)
+	}
```

Comment:
As a reference for ourselves. That :point_up: is a consequence of:

https://github.com/falcosecurity/libs/blob/2b2d06bf6411a7fe65377ee967358351541d3e91/userspace/libsinsp/plugin.cpp#L486-L493